Makefile

```diff
@@ -1,14 +1,18 @@
-.PHONY: all build clean run
+.PHONY: all build clean run network db
 
-all: build clean run
+all: build clean network db run
 
 build:
 	docker build -t collabtest/collabtest .
 
 clean:
-	-docker rm -f collabtest 2> /dev/null || true
+	-docker rm -f collabtest collabtestdb 2> /dev/null || true; docker network rm collabtest-network
```

Comment:
clean should nuke the db too. if implemented properly, it should be trivial to make a new db and we can always write a script to load test data or something.